### DISCUSSION TITLE
Inquiry on efficiently handling large volume of messages

### DISCUSSION THREAD

**Discussion by paeconsultoria** [devlikeapro/waha#463] [Q&A]
Hello,

We recently acquired an important client who requires sending 14,000 messages. Currently, we have set up a sending interval of 30 seconds to 1 minute, which would make this a very lengthy process to handle such a volume.

We were considering using broadcast lists, but we are concerned that WhatsApp might block the number, and we have not found this feature in Waha.

Could you suggest any other solutions to manage this volume of messages more efficiently?

Thank you very much.

**Comment by daviddr17** [comment]
Is a channel not an option for your, as users can subscribe to the channel and all of them are reading the message? Or do you need personal messages?

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by mmojadad** [reply]
What will be solution for personal messages?

**Reply by devlikepro** [reply]
I'd say WhatsApp Business official API or distributing the 14K messages across multiple accounts using WAHA.
Remember that even if you send messages to 14K customers and only few of them marked your message as spam - the account will be blocked anyway. It could happen even with "desired" payload, like pdf receipts/payment links, because people could change their number, they don't want to receive notifications anymore but there's no way to unsubscribe, etc

So consider not using main account for such things. Broadcast List from WhatsApp Business mobile app underthehood works the similar way, it just allow you to send to up to 256 clients at the same time (it's not supported in WAHA tho)

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
